charter-drafts/editing-2023.html

```diff
@@ -0,0 +1,582 @@
+            </dd>
+
+            <dt id="9706" class="spec">VirtualKeyboard API</dt>
+            <dd>
+              <p>The VirtualKeyboard API provides authors with greater control over the visibility of the virtual keyboard (VK), and greater ability to adapt the layout of web pages when VK visibility changes.</p>
+
+              <p class="draft-status"><b>Draft state:</b> <a href="https://www.w3.org/TR/virtual-keyboard/">Working Draft</a></p>
+              <p class="milestone"><b>Expected completion:</b> <abbr title="Candidate Recommendation">CR</abbr> Q1 2024 - Recommendation Q4 2024.</p>
+              <p><b>Adopted Draft:</b> VirtualKeyboard API, <a href="https://www.w3.org/TR/2022/WD-virtual-keyboard-20220505/">https://www.w3.org/TR/2022/WD-virtual-keyboard-20220505/</a>, 05 May 2022.
+              <p><b>Exclusion Draft:</b> <a href='https://www.w3.org/TR/2021/WD-virtual-keyboard-20210824/'>https://www.w3.org/TR/2021/WD-virtual-keyboard-20210824/</a>, 24 August 2021. <a href="https://www.w3.org/mid/cfe-8474-9fe7e748e12679045723d4afb052fe6ebf8ac5ff@w3.org">Exclusion period</a> began on 2021-08-24; ended on 2022-01-21.</p>
+              <p><b>Exclusion Draft Charter:</b> <a href='https://www.w3.org/2021/06/web-editing-wg-charter.html'>https://www.w3.org/2021/06/web-editing-wg-charter.html</a>.</p>
+            </dd>
+
+            <dt id="11707" class="spec">Clipboard API and events</dt>
+            <dd>
+              <p>This document describes APIs for accessing data on the system clipboard. It provides operations for overriding the default clipboard actions (cut, copy and paste), and for directly accessing the clipboard contents.</p>
+
+              <p class="draft-status"><b>Draft state:</b> <a href="https://www.w3.org/TR/clipboard-apis/">Working Draft</a></p>
+              <p class="milestone"><b>Expected completion:</b> <abbr title="Candidate Recommendation">CR</abbr> Q3 2024 - Recommendation Q3 2025.</p>
+              <p><b>Adopted Draft:</b> Clipboard API and events, <a href="https://www.w3.org/TR/2023/WD-clipboard-apis-20230516/">https://www.w3.org/TR/2023/WD-clipboard-apis-20230516/</a>, 16 May 2023.
+              <p><b>Exclusion Draft:</b> Clipboard Operations for the Web 1.0: Copy, Paste, Drag and Drop. <a href='https://www.w3.org/TR/2006/WD-clipboard-apis-20061115/'>https://www.w3.org/TR/2006/WD-clipboard-apis-20061115/</a>, 15 November 2006. <a href="http://lists.w3.org/Archives/Member/member-webapi/2006Nov/0009.html">Exclusion period</a> began on 2006-11-16; ended on 2007-04-15.</p>
+              <p><b>Exclusion Draft Charter:</b> <a href='http://www.w3.org/2006/webapi/admin/charter'>http://www.w3.org/2006/webapi/admin/charter</a>.</p>
+            </dd>
+
+            <dt id="11709" class="spec">Selection API</dt>
+            <dd>
+              <p>The EditContext is a new API that allows authors to more directly participate in the text input process.</p>
```

Comment:
Possible copy/paste issue, this line should talk about Selection API.

Reply:
Nice catch! Thank you @dandclark .